src/tiled/mainwindow.h

```diff
@@ -109,6 +109,7 @@ public slots:
     void dropEvent(QDropEvent *) override;
 
 private slots:
+    //File tab
```

Comment:
This would need to be "File menu" and there should be a space after `//`. But I don't really see a reason to have these comments.